djl-serving/workflows/multi-model/README.md

```diff
@@ -0,0 +1,285 @@
+                                                .fromInputStream(
+                                                        new ByteArrayInputStream(
+                                                                imageData.getAsBytes()));
+                                DetectedObjects detectedObjects =
+                                        gson.fromJson(
+                                                detectData.getAsString(), DetectedObjects.class);
+                                List<DetectedObjects.DetectedObject> items =
+                                        detectedObjects.items();
+                                for (DetectedObjects.DetectedObject item : items) {
+                                    if ("person".equals(item.getClassName())) {
+                                        Rectangle rect = item.getBoundingBox().getBounds();
+                                        int width = image.getWidth();
+                                        int height = image.getHeight();
+                                        Image sub =
+                                                image.getSubImage(
+                                                        (int) (rect.getX() * width),
+                                                        (int) (rect.getY() * height),
+                                                        (int) (rect.getWidth() * width),
+                                                        (int) (rect.getHeight() * height));
+                                        try (ByteArrayOutputStream bos =
+                                                new ByteArrayOutputStream()) {
+                                            sub.save(bos, "png");
+                                            output.add(bos.toByteArray());
+                                        }
+                                        output.addProperty("Content-Type", "image/png");
+                                        break;
+                                    }
+                                }
+                                return new WorkflowExpression.Item(output);
+                            } catch (IOException e) {
+                                throw new RuntimeException(e);
+                            }
+                        });
+    }
+}
+```
+
+Once it is done, you can package the whole project in a jar:
+
+```
+cd function/
+./gradlew jar
+```
+
+Then copy the jar in the `deps` folder under the model server home. For example, if the model server home is
+`/opt/djl/`, you can copy the jar to `/opt/djl/deps`.
+
+```
+cp build/libs/function-1.0-SNAPSHOT.jar /opt/djl/deps
+```
+
+Once all the external definitions are defined, the actual workflow definition can be created.
+We define the workflow.json to define the workflow.
+
+```
+{
+  "name": "multi-model",
+  "version": "0.1",
+  "models": {
+    "objectDetection": {
+      "application": "cv/object_detection",
+      "modelUrl": "djl://ai.djl.pytorch/ssd",
```

Comment:
Better use MXNet ssd model

Reply:
What's the reason?

Reply:
Avoid using two engines at the same time.

Reply:
Changed.